test/controllers/foo_controller_test.rb

```diff
@@ -0,0 +1,3 @@
+begin 
+rescue Exception
```

Comment:
__[staging][rubocop]__
* Do not suppress exceptions. [:ref](https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions)
* Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`? [:ref](https://github.com/bbatsov/ruby-style-guide#no-blind-rescues)

<!-- Comment by [SideCI](https://sideci.com) -->